fauna/data/initialize_communities_counter.js

```diff
@@ -0,0 +1,12 @@
+const CommunityInitialCounter = {
+  collection: 'CommunitiesCounter',
+  index: 'communities_counter',
+  key: 'counter',
+  data: [
+    {
+      counter: 0,
```

Comment:
Wondering, do we need to initialize these with the right count? Or am I missing how we’d make the state of the counters match the existing number of members/communities in the DB at present?

Reply:
OH... this is `TBD` in the main comment... seems like we should figure it out... possible to do as a migration?

Reply:
Done, I just added the migration along with instructions on how to run it :)

Reply:
Coolness!